### ISSUE TITLE
Digitally signing Windows installer

### ISSUE THREAD

**Issue by WhatDoesTheFoxSay** [radiant-rstats/radiant#137]
Since Radiant is a rare download from a 'stats' perspective (pun intended?), it gets caught by the Windows 10 SmartScreen feature (and Defender). If the app were digitally signed, this hinderance could be minimized.

**Comment by vnijs** [comment]
Fair point. Our schools IT created the windows installer and, apparently, getting it digitally signed is a bit of a hassle. You obviously don't have to use the all-in-one-installer but we do get a *lot* fewer questions and comments students since we started using it. If you know how to get something digitally signed and would be willing to contribute I assume I can get the installer script put on github as well.
